test/e2e/e2e_test.go

```diff
@@ -125,6 +125,51 @@ var _ = ginkgo.Describe("leaderWorkerSet e2e tests", func() {
 		testing.ExpectLeaderWorkerSetAvailable(ctx, k8sClient, lws, "All replicas are ready")
 	})
 
+	ginkgo.It("Can deploy lws with subgroupsize set", func() {
```

Comment:
add a test case, where "Deploy lws with subgroup size set and then update the group size"